internal_packages/composer/lib/send-action-button.jsx

```diff
@@ -116,19 +116,24 @@ export default class SendActionButton extends React.Component {
     return {preferred, rest};
   }
 
-  _contentForAction = ({iconUrl}) => {
-    let plusHTML = "";
-    let additionalImg = false;
+  _contentForAction = ({buttonTitle, title, iconUrl}) => {
+    let additionalImg = null;
+
+    const actionTitle = buttonTitle || title;
 
     if (iconUrl) {
-      plusHTML = (<span>&nbsp;+&nbsp;</span>);
-      additionalImg = (<RetinaImg url={iconUrl} mode={RetinaImg.Mode.ContentIsMask} />);
+      additionalImg = (
+        <span className="additional">
+          &nbsp;
+          <RetinaImg url={iconUrl} mode={RetinaImg.Mode.ContentIsMask} />
+        </span>
+      );
     }
 
     return (
       <span>
         <RetinaImg name="icon-composer-send.png" mode={RetinaImg.Mode.ContentIsMask} />
-        <span className="text">Send{plusHTML}</span>{additionalImg}
```

Comment:
`actionTitle` will default to the `title` ([line 122](https://github.com/nylas/N1/pull/2960/files#diff-efe6eb7a1b18b8d286bd0e97c9a438e9R122)), which is mandatory.
The title was already defined for the `SendActionButton` component here: [internal_packages/composer/lib/send-action-button.jsx#L47](https://github.com/nylas/N1/blob/a5ff3cfc6216b0c5eb9fbbdec504992bb4cbb7c7/internal_packages/composer/lib/send-action-button.jsx#L47) but was not used for the button.